style.module.scss

```diff
@@ -36,45 +91,32 @@
   }
 }
 
-.overviewInstallChoiceStep {
-  img {
-    max-width: 100px;
-    height: 60px;
-    margin: 0;
-    padding: 5px 0 5px 0;
-    border: none;
-    order: 1;
-  }
-
-  img, strong {
-    align-self: center;
-    text-align: center;
-  }
-}
-
 .overviewFeatureChoiceStep {
   flex-direction: column;
 }
 
-@media (min-width: 620px) {
-  .overviewInstallChoiceStep img {
-    order: 0;
-  }
+@media (min-width: 640px) {
   .overviewInstallChoice, .overviewFeatureChoice {
     margin-bottom: 10px;
   }
   .overviewInstallChoice {
-    grid-template-columns: 1fr 1fr 1fr 1fr;
-  }
-  .overviewInstallChoiceThree {
-    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr;
   }
   .overviewFeatureChoice {
     grid-template-columns: 1fr 1fr;
   }
   .overviewInstallChoiceStep, .overviewFeatureChoiceStep {
     flex-direction: column;
   }
+  .overviewInstallChoiceProviderText {
+    min-height: 38px;
```

Comment:
[note for reviewers] I only have min-height here, which will be applied to 2 or 3 buttons in a row case. For the 1 button in a row case, I found that 38px height here was too much space (need to scroll a lot in mobile, for example). 19px height looks good to me for 1 button in a row case, so I tweaked here.

Reply:
I'm not sure if this is the right change I'm commenting on, but if it is: I wonder if this wouldn't be better without the `min-height`, to reduce the whitespace:

<img width="830" alt="Screenshot 2024-04-02 at 9 55 22 PM" src="https://github.com/pganalyze/pganalyze-docs/assets/7227/26f370e2-8608-454d-ade9-77cd8ecdf51f">

Reply:
Actually, I just realized this is related to the earlier comment by you asking re: line-height vs min-height and I'm late to comment on that - this is not a strong opinion, so feel free to merge as-is (we can revise in a follow-up PR).

Reply:
Yeah let's go with as-is, happy to adjust later 👍
Also Jens haven't really checked the latest version, so he may want to say something about it too, which I can adjust as a follow-up if needed